emu/src/render/ppu.rs

```diff
@@ -62,15 +62,14 @@ impl PixelProcessUnit {
             }
             3 => {
                 // Bitmap mode
-                for y in 0..LCD_HEIGHT {
-                    for x in 0..LCD_WIDTH {
-                        let index_color = (y * LCD_WIDTH + x) * 2;
+                for x in 0..LCD_HEIGHT {
+                    for y in 0..LCD_WIDTH {
+                        let index_color = (x * LCD_WIDTH + y) * 2;
```

Comment:
I remember we had a discussion about this and @gallottino said that in the "graphics world" `y` is used for height and `x` for width

Reply:
Mmmm can be true, but is only temporary, this will be probably changed :)